docs/src/main/sphinx/object-storage/file-system-azure.md

```diff
@@ -49,6 +49,29 @@ system support:
     you encounter rate limiting issues.
 :::
 
+(user-assigned-managed-identity-authentication)=
+## User-Assigned Managed Identity authentication
```

Comment:
```suggestion
## User-assigned managed identity authentication
```

Reply:
Do we really need the "User-assigned" part .. seems redundant and also you are not using it in the config name.. that makes it look redundant and at least inconsistent